load/S3Utility.py

```diff
@@ -70,7 +70,7 @@ def uploadFileset(self, s3Bucket, inputFileset):
 			return False
 		else:
 			if inp.filesetPrefix.startswith("video"):
-				TranscodeVideo.transcodeVideoFileset(self.config, inp.filesetPrefix, inp.s3FileKeys())
+				# TranscodeVideo.transcodeVideoFileset(self.config, inp.filesetPrefix, inp.s3FileKeys())
```

Comment:
this was commented out. is there a reason?

Reply:
@bradflood , I fixed it as it was a mistake. It can not be commented.